packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -135,6 +149,20 @@ public void onMethodCall(MethodCall methodCall, Result result) {
     }
   }
 
+  private static String initialParametersToString(Map<String, Object> parameters)
```

Comment:
Expecting a Map -> `application/x-www-form-urlencoded` encoding in each platform implementation sounds a little fragile in case these implementations don't exactly match.

My preference would be to take a similar approach to Android's WebView and WkWebView where the encoding is the responsibility of the app developer, which allows some more flexibility for developers.

Reply:
I think this is how you're suppose to do that, webview send form by post with this.